specification/ai/Azure.AI.Projects/common/models.tsp

```diff
@@ -0,0 +1,140 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+
+namespace Azure.AI.Projects;
+
+@doc("The available sorting options when requesting a list of response objects.")
+union ListSortOrder {
+  string,
```

Comment:
What other orders would you expect to add? And since this is an input only enum, I don't think making it extensible is needed...